elephant/spike_train_dissimilarity.py

```diff
@@ -276,11 +280,12 @@ def _victor_purpura_dist_for_st_pair_intuitive(
             for j in range(1, nspk_b+1):
                 scr[i, j] = min(scr[i-1, j]+1, scr[i, j-1]+1)
                 scr[i, j] = min(scr[i, j], scr[i-1, j-1] + np.float64((
-                               q*abs(train_a[i-1]-train_b[j-1])).simplified))
+                                                                              cost_factor * abs(spiketrain_a[i - 1] - spiketrain_b[j - 1])).simplified))
     return scr[nspk_a, nspk_b]
 
 
-def van_rossum_dist(trains, tau=1.0 * pq.s, sort=True):
+@deprecated_alias(trains='spiketrains')
+def van_rossum_dist(spiketrains, tau=1.0 * pq.s, sort=True):
```

Comment:
Can't `tau` be replaced by `time_constant`? In Van Rossum (2001), the formula does not have `tau`. It mentions `t_c` in the formula for the parameter of the exponential function. Therefore, not only we should rename this to the meaningful parameter, but also make the proper formula reference in the Notes.

Also, the function could be renamed to `van_rossum_distance`.

Reply:
I would agree to rename to `time_constant` in this case, making the reference to t_c is also a good idea.

Reply:
Renamed to van_rossum_distance, replaced arg name by time_constant, and added a reference to the paper.